Classes/Middleware/GetAuthenticationCode.php

```diff
@@ -68,10 +68,12 @@ public function process(ServerRequestInterface $request, RequestHandlerInterface
      */
     protected function isInstagramAuthentificationRedirect(ServerRequestInterface $request): bool
     {
-        if (!empty($request->getQueryParams()['code'])) {
-            $code = $request->getQueryParams()['code'];
-            if (strlen($code) > 8) {
-                return true;
+        if (in_array('https://l.instagram.com/', $request->getHeader('referer'))) {
```

Comment:
In my case the referer header contains the App return URL and not l.instagram.com - not sure if instagram changed anything in the way they redirect. With this check, no authentication code can be obtained.